Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -372,8 +372,8 @@ private bool TryGenerateActivationPoint(IMixedRealityHand jointedHand, out Vecto
                     break;
                 case SolverSafeZone.UlnarSide:
                 default:
```

Comment:
This also changes the `default` case. Is that intended?

Reply:
yes. the default is ulnar